include/lsst/jointcal/PhotometryTransform.h

```diff
@@ -224,7 +224,7 @@ class PhotometryTransformChebyshev : public PhotometryTransform {
      * @param[in]  bbox   The bounding box it is valid within, to rescale it to [-1,1].
      * @param[in]  identity If true, set a_0,0==1, otherwise all coefficients are 0.
      */
-    PhotometryTransformChebyshev(size_t order, afw::geom::Box2D const &bbox, bool identity);
+    PhotometryTransformChebyshev(size_t order, lsst::geom::Box2D const &bbox, bool identity);
```

Comment:
The only reason we use `lsst::geom` instead of just `geom::` in afw is because *within* `lsst::afw::`, `geom::` is ambiguous (because `lsst::afw::geom` also exists).  That's not a problem in `lsst::jointcal`, so you should just use `geom::` here.

Reply:
I had thought I was getting warnings at one point when I tried that, but I don't see them now.